FakeHttpClient/FakeHttpClient/RuleResponseHeadersExtension.cs

```diff
@@ -6,13 +6,31 @@ public static class RuleResponseHeadersExtension
     {
         public static Rule UseAcceptRangeInResponse(this Rule rule, string range)
         {
-            rule.AddModifier(x => x.Headers.AcceptRanges.Add(range));
+            rule.AddModifier((request, response) => response.Headers.AcceptRanges.Add(range));
             return rule;
         }
 
         public static Rule UseTransferEncodingInResponse(this Rule rule, string encoding)
```

Comment:
SetTransferEncodingInResponse

//Use doens't seem that readable, we'll try with Set?
//The comment is applicable in other places too...

You could merge this, and then fix the comment in the next change request